apps/web/package.json

```diff
@@ -8,6 +8,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "dev:test": "playwright test",
+    "e2e:test": "start-server-and-test \"npm run start\" http://localhost:3000 \"playwright test\"",
```

Comment:
I don't mind `dev:test` but for the other I think we'd want `test:e2e` so you could then have `test:unit` and `test:integration` while `test` would run all three 😄